tests/test_weight.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
The tests are insufficient.
You should add a test for checking what is the target weight, how much to go and so on.
also, I would suggest checking if the weight is displaying correctly.

Reply:
I added tests